src/util.h

```diff
@@ -51,6 +51,13 @@ extern bool fLogIPs;
 extern volatile bool fReopenDebugLog;
 extern CTranslationInterface translationInterface;
 
+extern const char * const BITCOIN_CONF_FILENAME;
+extern const char * const BITCOIN_PID_FILENAME;
+
+static const bool DEFAULT_SELFSIGNED_ROOTCERTS = false;
+static const bool DEFAULT_CHOOSE_DATADIR = false;
```

Comment:
It's ugly to have defaults for these GUI specific options in the core, they are not related to util at all.
IMO the help for GUI specific options should be moved to the GUI part of the code.